### DISCUSSION TITLE
How  to  disable comment formatting?

### DISCUSSION THREAD

**Discussion by ShindouMihou** [facebookincubator/ktfmt#447] [Q&A]
It seems like `ktfmt` struggles with formatting comments a **INSANE TON** which creates an incredible amount of stink in the code. Is there any way to stop it from formatting comments at all?

![image](https://github.com/facebook/ktfmt/assets/69381903/21329e32-9a02-449b-918b-57c46a3643e5)

**Comment by hick209** [comment]
Unfortunately there isn't and that's not something that we have plans to implement
